opencti-platform/opencti-graphql/src/modules/organization/organization-resolver.ts

```diff
@@ -36,6 +37,10 @@ const organizationResolvers: Resolvers = {
     subOrganizations: (organization, _, context) => subOrganizationsLoader.load(organization.id, context, context.user),
     parentOrganizations: (organization, _, context) => parentOrganizationsLoader.load(organization.id, context, context.user),
     default_dashboard: (current, _, context) => loadByIdLoader.load(current.default_dashboard, context, context.user),
+    grantable_groups: (organization, _, context) => findGrantableGroups(context, context.user, organization),
```

Comment:
I think it needs to be batched, otherwise organization list will be very slow to load. and we also get roles in eatch groups, which needs to be batched too (group resolver does that, but here we don't)

Reply:
Oh I see that SettingsOrganizationLine_node doesn't try to get grantable_groups, it was just slow for me locally, not sure then if it's necessary